.dockerignore

```diff
@@ -0,0 +1,2 @@
+.git
+.cache
```

Comment:
The `.dockerignore` file ignores _host_ files, i.e. those in the repo. Is `.cache` created in a previous `RUN` step somehow? I think we could exclude most dot directories, maybe with `.*/`.

Reply:
I also think we could ignore `node_modules/` here. That would save some more time and the `Dockerfile` installs the dependencies anyway.

Reply:
Ah, I was actually not sure about `.cache`. I took that from the ignore file for `addons-server`.

> I also think we could ignore node_modules/ here. That would save some more time and the Dockerfile installs the dependencies anyway.

So, adding that to `.dockerignore` would remove `/srv/code/node_modules/`. Are you saying that is unused and only `src/node/node_modules/` is needed?

Reply:
Yes. `/srv/code/` is a copy of the repo (minus the ignored files listed in the `.dockerignore`).

On Circle, we might not have a `node_modules/` directory actually and that might explain the difference you noticed between your locally-built image and the ones from the Docker Hub.

Reply:
> On Circle, we might not have a node_modules/ directory actually and that might explain the difference you noticed between your locally-built image and the ones from the Docker Hub.

You are right.

> I think we could exclude most dot directories, maybe with .*/.

That would eliminate less than 1MB, and I am not sure if it would break anything, so maybe not worth it?